test/default.nix

```diff
@@ -1,10 +1,10 @@
-let
-  haskellNix = (import ../default.nix {});
-in
-{ pkgs ? import nixpkgs nixpkgsArgs
-, nixpkgs ? haskellNix.sources.nixpkgs-default
+{ haskellNix ? import ../default.nix { inherit checkMaterialization; }
+, pkgs ? import nixpkgs nixpkgsArgs
+, nixpkgs ? haskellNix.sources.nixpkgs
 , nixpkgsArgs ? haskellNix.nixpkgsArgs
 , ifdLevel ? 1000
+, compiler-nix-name ? "ghc865"
```

Comment:
Does this need to be the same as any of the other versions? When can we upgrade it?